line_profiler/line_profiler.py

```diff
@@ -420,12 +422,29 @@ def main():
 
     parser = optparse.OptionParser(usage=usage,
                                    version=__version__)
+    parser.add_option('-k', '--skip', action='store_true', dest='skip',
+        default=False, help="skip displaying functions that have no runtime")
+
+    parser.add_option('-u', '--unit', default=None,
+                      help="specify the unit of time")
 
     options, args = parser.parse_args()
     if len(args) != 1:
         parser.error("Must provide a filename.")
+        
+    output_unit = None
+    try:
+        if options.unit is not None:
+            time_unit = float(options.unit)
+            if time_unit != 0:
+                output_unit = time_unit
+    except:
+        pass
+    
     lstats = load_stats(args[0])
-    show_text(lstats.timings, lstats.unit)
+    skip = True if options.skip else False
```

Comment:
Please change: 

```python
skip = True if options.skip else False
```
![image](https://user-images.githubusercontent.com/3186211/97095143-db152b80-1629-11eb-9f87-ed6fc73eec27.png)

----

to: 

```python
skip = options.skip
```
![image](https://user-images.githubusercontent.com/3186211/97095139-d486b400-1629-11eb-8644-5a120d72145b.png)

----

or 

```python
skip = bool(options.skip)
``` 

to have exact equivalence to the original code, but the value of `options.skip` should always be a bool, so there is no need for that. Having the if-else statement is superfluous.

Reply:
ill use bool(options.skip) because it needs to parse None as False aswell